### PR TITLE
Tools: Bump lint.sh version

### PR DESCRIPTION
**Pull request by spycrab**
Fixes the linter not working due to the buildbot version bump.

### CODE REVIEW

**Comment by spycrab** [comment]
@lioncash Done.

File: Source/Core/VideoBackends/OGL/FramebufferManager.cpp [hunk 1]
```diff
@@ -263,23 +263,27 @@ FramebufferManager::FramebufferManager(int targetWidth, int targetHeight, int ms
                 "vec4 sampleEFB(ivec3 pos) {\n"
                 "	vec4 color = vec4(0.0, 0.0, 0.0, 0.0);\n"
                 "	for(int i=0; i<" +
-                samples.str() + "; i++)\n"
-                                "		color += texelFetch(samp9, pos, i);\n"
-                                "	return color / " +
-                samples.str() + ";\n"
-                                "}\n";
+                samples.str() +
+                "; i++)\n"
```
Comment (phire) [inline Source/Core/VideoBackends/OGL/FramebufferManager.cpp]: This one was already mangled by a previous formatting, but should probally be fixed too.

**Comment by Zexaron** [comment]
**For future reference:** 

This PR bumps the required clang-format to version 5.0.0 - since it's not that obvious on the first glance.

LLVM docs took a bit to dig through, since clang-format is just a little standalone utility part of a huge package, I put the docs thorugh diff comparison, if the docs even contain all the changes, (probably not all configurable so maybe not in the styles config doc) then 6.0.0 has these 3 new options:

- _IncludeBlocks_
- _IndentPPDirectives_
- _RawStringFormats_

clang-format 7.0.0 has this over 6.0.0 (nonfinalized doc):

- _Clang-format will now support detecting and formatting code snippets in raw string literals. This is configured through the RawStringFormats style option._

However it's possible in VS 15.7 now to also specify a user custom `clang-format.exe` officially right in the in the same VS Settings window below the default formatting style, it's not shown in the preview articles, but looks like it was part of 15.7 since I have it right here.

Warning one of those crazy ideas ahead - no need to continue reading.

> Here's a thought: Maybe the differences the people see or will if they use a newer clang-format version like 6.0.0 or the coming 7.0.0, could be mitigated without having to bump the version and do a similar PR like this. By simply putting the new style config options into the _.clang-format_ file for those version which those newer version would respect, while the older 5.0.0 version would hopefully just ignore those lines and continue respecting the rest as before, if this is actually worth it. I'm just thinking it may be if it's not desirable to diverge the version off what VS uses, while , that might relieve some people not having to use another clang format specifically for dolphing if they have to use other versions for other projects already. I might actually go test this theory if I feel one day. That said, I agree with keeping it in-sync with what VS2017 uses going forward.

**Comment by Zexaron** [comment]
VS2017 15.8 bumped the shipped clang-format.exe to 6.0.0 - just a heads up (even tho I'm 3 days late)

**Pull request merged**